lib/data/2022_hud_lists.json

```diff
@@ -883,18 +883,6 @@
         "key": 34,
         "description": "Rental by client in a public housing unit"
       },
-      {
-        "key": 35,
-        "description": "Staying or living in a family member's room, apartment or house"
-      },
-      {
-        "key": 36,
-        "description": "Staying or living in a friend's room, apartment or house"
-      },
-      {
-        "key": 37,
-        "description": "Worker unable to determine"
-      },
```

Comment:
These were incorrectly being included in the "destinations" list. They are not valid destinations in 2022. This will affect any usage of `HudUtility.destination` and `HudUtility.destinations`. is this OK?

Reply:
I think that should be fine.  Good catch!